src/directions/types.ts

```diff
@@ -265,6 +265,19 @@ export interface MapLibreGlDirectionsConfiguration {
    * ```
    */
   dragThreshold: number;
```

Comment:
Would be nice to update the `dragThreshold`'s docs as well, to indicate that this option also affects the behavior when your option is enabled.

Reply:
done